simple-analytics.php

```diff
@@ -17,25 +17,39 @@
  * @since 1.0.0
  */
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit;
-}
+namespace SimpleAnalytics;
 
-function simpleanalytics_noscript() {
-	$domain = get_option( 'simpleanalytics_custom_domain' ) ?: 'queue.simpleanalyticscdn.com';
+defined('ABSPATH') || exit;
 
-	echo '<noscript><img src="https://' . $domain . '/noscript.gif" alt="" referrerpolicy="no-referrer-when-downgrade"></noscript>' . "\n";
+function should_collect_analytics(): bool
```

Comment:
Nice work already :) I know it's a draft but..

Don't forget that the is_logged_in is not the only thing that effects should_collect_analytics. People should be able to turn that behavior off via the UI. Default is on.

Reply:
Something I found online:

![2024-03-05-15h43-screenshot@2x](https://github.com/simpleanalytics/wordpress-plugin/assets/1079135/dfdcf461-f4d8-4be1-b055-00ef1b7e7c29)

The IPs are a nice to have.

Reply:
Thanks! Will include it in the "advanced" tab